templates/collectd.conf.j2

```diff
@@ -37,4 +37,11 @@ LoadPlugin netlink
   QDisc "eth1"
 </Plugin>
 
+LoadPlugin disk
+<Plugin "disk">
+  Disk "sdb"
```

Comment:
Also, please add  the default variable `aerospike_collectd_disks` to the README.md for documentation.